### DISCUSSION TITLE
Welcome to MLNetExamples Discussions!

### DISCUSSION THREAD

**Discussion by jwood803** [jwood803/MLNetExamples#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sharpninja** [comment]
Can you add a README.md for each project explaining the example?

**Reply by jwood803** [reply]
Great idea! I'll work on that 😄
